app/src/main/java/com/github/libretube/ui/fragments/PlayerFragment.kt

```diff
@@ -181,6 +181,7 @@ class PlayerFragment : Fragment(), OnlinePlayerOptions {
      * SponsorBlock
      */
     private var segments = listOf<Segment>()
+    private var catConfig = PlayerHelper.getSponsorBlockCategories()
```

Comment:
Please name that var `sponsorBlockConfig` and init it after `sponsorBlockEnabled` for better readability